cost-analyzer/values.yaml

```diff
@@ -181,6 +181,7 @@ kubecost:
 kubecostModel:
   image: "gcr.io/kubecost1/cost-model"
   imagePullPolicy: Always
+  outOfClusterEnabled: false
```

Comment:
Looks good, but I propose we make this name a tad more specific. We often refer to outOfCluster as a broad group of functionality. Something like `outOfClusterPromEnabled` or `outOfClusterPromMetricsEnabled` could work?

@AjayTripathy or @michaelmdresser, you may have much better ideas!

Reply:
outOfClusterPromMetricsEnabled seems the right level of specificity, since we have a few different ways of getting/reading out of cluster metrics.

Reply:
Updated.